util/start_test

```diff
@@ -89,6 +89,7 @@
 #   -memleaks
 #   -no-chpl-home-warn                Do not show warning when start_test invoked outside CHPL_HOME
 #   -progress                         Print pass/fail to stderr for each test. Does not work for directories.
+#   -test-root                        Absolute path to test dir. Set when test dir is not under CHPL_HOME.
```

Comment:
Might consider calling this -test-root-dir or -root-test-dir. I don't feel strongly about this, but I initially thought this meant test as root